Common/src/main/java/com/tabnineCommon/binary/requests/login/LoginRequest.kt

```diff
@@ -3,12 +3,16 @@ package com.tabnineCommon.binary.requests.login
 import com.tabnineCommon.binary.BinaryRequest
 import com.tabnineCommon.binary.requests.EmptyResponse
 
-class LoginRequest : BinaryRequest<EmptyResponse> {
+class LoginRequest(private val onSuccessFn: ((EmptyResponse) -> Unit)? = null) : BinaryRequest<EmptyResponse> {
```

Comment:
You don't need to pass the empty response here, just make it accept 0 args

Reply:
ok